src/stdio.rs

```diff
@@ -3,7 +3,7 @@ use std::os::unix::io::{AsRawFd, RawFd};
 use anyhow::Result;
 use nix::unistd::dup2;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
It doesn't seem FileDescriptor is used in any kind of new way. Is there a reason for it that I'm not seeing? I'm sure a clone of a RawFd isn't an issue since it's just an i32, but I'm wondering if this trait is ever actually used.

Reply:
This is needed to clone the `console_socket`, which is `Option<FileDescriptor>`. I need to pass this into the `container_init` function, which runs inside the new process. Unlike fork where everything is duplicated behind scenes, here we need to clone the memory explicitly.